presto-phoenix/src/test/java/io/prestosql/plugin/phoenix/TestPhoenixIntegrationSmokeTest.java

```diff
@@ -157,12 +157,6 @@ public void testCreateSchema()
         throw new SkipException("test disabled until issue fixed"); // TODO https://github.com/prestosql/presto/issues/2348
     }
 
-    @Override
-    public void testDropSchema()
-    {
-        throw new SkipException("test disabled until issue fixed"); // TODO https://github.com/prestosql/presto/issues/2348
```

Comment:
the related issue is not fixed.

Reply:
This is moved to `TestPhoenixDistributedQueries`